packages/markdown-oxide/brioche.lock

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "rust": "1d19b3a0b14ba3a7abef2d061287fa4ccd85705ad1ca9483d78ed31a494698e5",
+    "std": "234178677be46196ab837fb232fa4e9a3a77eb751385777e35b979c79a161bfd"
+  },
```

Comment:
Small fix for the lockfile:

```suggestion
  "dependencies": {},
```

(This can happen if you write a project outside `brioche-packages` then move it into `brioche-packages` due to the way [workspaces](https://brioche.dev/docs/core-concepts/workspaces/) work... not sure if there's a good fix for this, but not the first time this has happened so I might try to mention this in the contribution guide at least!)